CDCgov/prime-reportstream
# 1214 Migrate the quick tests to run as part of the integration tests

I have been slowly working on this over a long period of time.  This PR converts the quick tests from a shell script using the PRIME CLI to a set of integration tests in Kotlin.  The ProcessData class, which is the one used for the PRIME CLI data process is reused programmatically and I added the reuse of the metadata object, so it runs quicker.  With this change the quick tests will also run as part of the build.

Test steps:
1. Run the integration tests using `./gradlew testIntegration` and verify the quick tests run successfully. 

## Changes
- Migrated the quick tests to the integration test suite.

## Checklist

### Testing
- [ ] Ran `quickTest all`?  <------ THIS GOES AWAY!!!!!!

### Process

## Fixes
- #1214

## To Be Done
